dist/main.js

```diff
@@ -0,0 +1,64 @@
+const game = new GoldRush()
+const Render = new Renderer()
+const UP = 38
```

Comment:
nice I like that you added these constants. Makes it easier to change/edit in the future